### DISCUSSION TITLE
Installation issue in Windows 10

### DISCUSSION THREAD

**Discussion by yuhsuanho02** [open-edge-platform/anomalib#1955] [Q&A]
I am a beginner in AI, and I encountered some issues while installing anomalib. Can someone help me?

OS: Windows10
Python:3.10

I want to install from source.

![1](https://github.com/openvinotoolkit/anomalib/assets/97345804/48c32534-e462-474b-abc0-999a9e1d091c)

When I pip install -e . , It shows an error as in the following picture.

![2](https://github.com/openvinotoolkit/anomalib/assets/97345804/38c11d7a-093e-4927-b329-e21c5d486685)

Can someone help me? Thank you.

**Comment by blaz-r** [comment]
I've encountered a similar issue. It seems like jsonnet is not available for windows, and jsonnet-binary that used to be a binary distribution for windows is not available for python 3.10.
At this point, I'm not sure how to fix this.

**Comment by samet-akcay** [comment]
hm, I was not aware of this. Maybe we could raise this issue on [jsonargparse](https://github.com/omni-us/jsonargparse) repo. Until then we could remove `all` from `jsonargparse` requirements.
https://github.com/openvinotoolkit/anomalib/blob/09cbadea3c17d4352b115302ef0c40d426556145/pyproject.toml#L21

So if you could modify this line to `"jsonargparse>=4.27.7"`, it should work
